UIKitTests/DisplayLinkTests.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DisplayLinkTests.swift
+//  UIKitTests
+//
+//  Created by Michael Knoch on 26.06.18.
+//  Copyright © 2018 flowkey. All rights reserved.
+//
+
+import XCTest
+@testable import UIKit
+
+class DisplayLinkTests: XCTestCase {
+    var displayLink: DisplayLink!
+
+    override func setUp() {
+        DisplayLink.activeDisplayLinks = []
```

Comment:
Not sure it makes sense to test this because it's our own implementation. What do you think? Is this just for code coverage?